### PR TITLE
[feature] External Slurmdbd entrypoint setup

### PR DESCRIPTION
**Pull request by hehe7318**
### Description of changes
* Setup External Slurmdbd entrypoint
* Mock necessary attributes
* Include necessary recipes

### Tests
* Not finished yet

### References
* CDK PR: https://github.com/aws/aws-parallelcluster/pull/5888

### Checklist
- Make sure you are pointing to **the right branch**.
- If you're creating a patch for a branch other than `develop` add the branch name as prefix in the PR title (e\.g\. `[release-3.6]`).
- Check all commits' messages are clear, describing what and why vs how.
- Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# TODO: move this template to a separate recipe
+template '/etc/systemd/system/slurmdbd.service' do
+  source 'slurm/head_node/slurmdbd.service.erb'
+  owner 'root'
+  group 'root'
+  mode '0644'
+  action :create
+end
+
+include_recipe "aws-parallelcluster-slurm::config_slurm_accounting"
```
Comment (jdeamicis) [inline cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb]: This must be the last thing in the entrypoint.

File: cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
```
Comment (jdeamicis) [inline cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb:14]: Please put a comment here saying that these are just _mocks_ needed to reuse the recipes from PC as they are.

**Review by jdeamicis — COMMENTED** [review]
Left a couple of comments.

File: cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb (hunk 2 above)
Comment (hehe7318) [reply cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb:14]: Done!

File: cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb (hunk 1 above)
Comment (hehe7318) [reply cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb]: Done!

**Review by jdeamicis — APPROVED** [review]

**Pull request merged**